src/config/androidconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "supportedRuntimes" : ["android-30", "android-29", "android-28", "android-27", "android-26", "android-25", "android-24", "android-23"],
+    "minSupportedAndroidRuntime" : "23",
```

Comment:
since we're no longer doing a string comparison here, I updated the value to only be the version number without `android-`. But am open to putting `android-` back as well.